protx/utils/demographics.py

```diff
@@ -2,10 +2,15 @@
 import json
 import numpy as np
 import pandas as pd
+from plotly.subplots import make_subplots
+import plotly.graph_objects as go
 from protx.utils.plotly_figures import timeseries_lineplot
 
 db_name = '/protx-data/cooks.db'
 
+six_colors_light_green_to_blue = ['#eff5d6', '#c6e8b0', '#8fcca1', '#62ad9c', '#3c7d8a', '#26547a']
```

Comment:
Can we consolidate these magic string values into one of the config file somewhere? 
I get a bad code smell mixing the styles(hexstring) values directly into the python code.

Reply:
We should create a styles.conf.py config file (or something like that) for all visual style and design info to be tracked for use consistently across all the plot components (and the map).
Especially if we collapse the "many views" into a singular view that handles all the use cases for map + plots.

Reply:
> We should create a styles.conf.py config file (or something like that)

good idea 💯